app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -166,6 +166,14 @@ def self.default_blacklisted_event_names
       )
     end
 
+    def self.update_alert(*args)
+      _log.debug("Updating an alert on Hawkular provider: #{args}")
```

Comment:
This method is left without implemented on purpose.
It is implemented on following PR.
It has some complexity which is good to split.